docs/lang/branch.md

```diff
@@ -82,15 +82,15 @@ switch (i) {
 
 ```cpp
 char i = 'A';
-// 这里的 i 的数据类型是字符型 ，但 char
-// 也是属于整数的类型，满足整数类型的表达式的要求
+// the data type of i here is character, but char
+// is also a type of integer and satisfies the requirements of an expression of integer type
```

Comment:
// the data type of i here is character, but it is a kind of integer in
// nature and satisfies the requirement of an integer-typed expression

Reply:
:thinking: Waiting for upstream discussion